src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests.cs

```diff
@@ -12,6 +12,7 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public partial class IOperationTests : SemanticModelTestBase
     {
+        [CompilerTrait(CompilerFeature.IOperation)]
```

Comment:
That is true for most of the files here, tomorrow if you add more tests for other scenario will we go remember to split it ?